cmd/celestia-appd/cmd/modify_root_command_multiplexer.go

```diff
@@ -14,13 +20,6 @@ import (
 	embedding "github.com/celestiaorg/celestia-app/v4/internal/embedding"
 )
 
-// v2UpgradeHeight is the block height at which the v2 upgrade occurred.
```

Comment:
This still needs to be set here eventually, so im thinking of bringing the chainID parsing logic from v3 here instead?

Reply:
That works. Sounds like we would un-deprecate the `--v2-upgrade-height` flag but instead of a user passing it on command line, they would pass it via the new file.

Reply:
yeah, so i totally removed it from v4 to avoid confusion.

Reply:
letting the user pass it manually via the file may be better than having default, especially that v3 already has default based on chain-id 👍🏾